internal/graph/check.go

```diff
@@ -884,18 +880,15 @@ func (c *LocalChecker) checkTTU(parentctx context.Context, req *ResolveCheckRequ
 				continue
 			}
 
-			if len(condEvalResult.MissingParameters) > 0 {
-				errs = multierror.Append(errs, condition.NewEvaluationError(
-					t.GetCondition().GetName(),
-					fmt.Errorf("tuple '%s' is missing context parameters '%v'",
-						tuple.TupleKeyToString(t),
-						condEvalResult.MissingParameters),
-				))
-
-				continue
-			}
-
 			if !condEvalResult.ConditionMet {
+				if len(condEvalResult.MissingParameters) > 0 {
+					errs = multierror.Append(errs, condition.NewEvaluationError(
+						t.GetCondition().GetName(),
+						fmt.Errorf("tuple '%s' is missing context parameters '%v'",
+							tuple.TupleKeyToString(t),
+							condEvalResult.MissingParameters),
+					))
+				}
 				continue
 			}
```

Comment:
Why don't we change this directly within `EvaluateTupleCondition()`? we have this in 4 places already, and in listUsers we're going to copy it again

Reply:
We definitely should do as suggested. I will incorporate that change if we agree to this change in principle.